src/rosdistro/__init__.py

```diff
@@ -49,6 +49,10 @@
     from io import BytesIO as StringIO
 import sys
 import yaml
+try:
+    from urllib.parse import urlparse
+except ImportError:
+    from urlparse import urlparse
```

Comment:
Please move this block on line higher (above `yaml`) to keep the import order alphabetically.